src/Datadog.Trace.Tools.Runner/Options.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+
+namespace Datadog.Trace.Tools.Runner
+{
+    internal class Options
+    {
+        [Usage(ApplicationAlias = "ddtrace")]
+        public static IEnumerable<Example> Examples
+        {
+            get
+            {
+                yield return new Example("Set CI environment variables", UnParserSettings.WithUseEqualTokenOnly(), new Options { SetEnvironmentVariables = true });
+                yield return new Example("Wrap a command with the CLR profiler environment variables", UnParserSettings.WithUseEqualTokenOnly(), new Options { Value = new string[] { "dotnet", "test" } });
+                yield return new Example("Wrap a command with the CLR profiler and change the datadog environment key", UnParserSettings.WithUseEqualTokenOnly(), new Options { Environment = "ci", Value = new string[] { "dotnet", "test" } });
+                yield return new Example("Wrap a command with the CLR profiler and changing the datadog agent url", UnParserSettings.WithUseEqualTokenOnly(), new Options { AgentUrl = "http://agent:8126", Value = new string[] { "dotnet", "test" } });
+            }
+        }
+
+        [Option("set-ci", Required = false, Default = false, HelpText = "Setup the clr profiler environment variables for the CI job. (only supported in Azure Pipelines)")]
```

Comment:
I would add a little more explanation that the --set-ci flag will not run any other code besides the configuration. Maybe the following?

```suggestion
        [Option("set-ci", Required = false, Default = false, HelpText = "Setup the clr profiler environment variables for the CI job and exit. (only supported in Azure Pipelines)")]
```